setup-script.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+OPAQUE_HOME=/home/vitikoo/my_code/opaque
```

Comment:
Do we need this script? I don't think it's referenced anywhere and it contains a hardcoded username so it won't work out of the box.

Reply:
Removed.